tools/packaging/kernel/build-kernel.sh

```diff
@@ -461,6 +474,13 @@ main() {
 			v)
 				kernel_version="${OPTARG}"
 				;;
+			x)
+				conf_guest="${OPTARG}"
+	      case "$conf_guest" in
+		      sev) ;;
```

Comment:
it'd be great if you can add a log, ie: " Enabling confidential guest protection: SEV"

Reply:
There is a similar log comment already added: 
```
215   if [[ "${conf_guest}" != "" ]];then$
216      info "Add kernel config for ${conf_guest} guest protection"$
```